tf_encrypted/keras/backend.py

```diff
@@ -0,0 +1,34 @@
+"""TFE Keras backend"""
+import tensorflow as tf
+from tensorflow.keras import backend as K
+
+import tf_encrypted as tfe
+from tf_encrypted.protocol.pond import PondPrivateVariable
+
+
+# TODO Should be moved outside backend, maybe in pond.py?
+def is_variable_initialized(x):
+  """Test if TFE private variable has been initialized"""
+  assert isinstance(x, PondPrivateVariable), type(x)
+  is_initialized = tf.is_variable_initialized(x.share0.variable)
+  return is_initialized
+
+
+def get_session():
+  """Returns the TFE session to be used"""
+  sess = K.get_session()
+  # If session is not a tfe.Session, creates one
+  # K.get_session() will return a tfe.Session if
+  # TFE global session was previously set with set_session
+  if not isinstance(sess, tfe.Session):
+    sess = tfe.Session()
```

Comment:
each consecutive `get_session` will create a new `tfe.Session`, not sure that's what we want. i'm not convinced we should mix, or try to reuse, the `tf.keras` session but rather manage our own. thoughts? cc @yanndupis @jvmancuso

Reply:
I agree.  See how tf.keras does it [here](https://github.com/tensorflow/tensorflow/blob/88340a942c897601cc4d31c7c61bf68f60446857/tensorflow/python/keras/backend.py#L91) and [here](https://github.com/tensorflow/tensorflow/blob/88340a942c897601cc4d31c7c61bf68f60446857/tensorflow/python/keras/backend.py#L422)